### ISSUE TITLE
Keys Activated Randomly

### ISSUE THREAD

**Issue by alextechtai** [iseekwonderful/csgoGlow#282]
Not sure if it's the script itself or it's conflicting with one of my macOS utilities, but my command key or other keys on my keyboard gets activated at random times when the button isn't pressed. I have to manually go to the accessibility on screen keyboard to deactivate.

**Comment by amcgregor** [comment]
Indeed, can confirm modifier keys appear to get stuck.  Haven't isolated the condition or situation that encourages that to happen, but I suspect it's related to ⌘⇥ (switching away) from the game, then back.  E.g. this seems to happen reliably to me when switching between the game and Textual 6 IRC client.  Usually it's ⌃ (control), which at least doesn't prevent me from getting back to Terminal to type `c` (⌃C) to kill it, freeing the stuck key.

Edited to add: beyond every key being control+that key, in-game the result is that I lose the ability to crouch (bound to control).

**Comment by alextechtai** [comment]
@amcgregor Could it be possible it's one of the mac utilities/apps we have running in the background? If so what are ones you're running?

**Comment by ethano8225** [comment]
I assumed it was just my old mac that was acting up, and not a problem with the cheat itself. Thanks for telling me that I can do the on-screen keyboard trick though, as restarting my pc every time this happens is a bit much lol.

**Comment by alextechtai** [comment]
@etan8225 I actually thought it was my Macbook Pro keyboard malfunctioning LOL I was almost going to pay $$ to have my entire top case replaced because Apple won't just replace the keyboard... Now I'm content with its current conditions. I do want to figure out what is triggering the toggling of the keys though...

**Comment by amcgregor** [comment]
Question of the day: does anyone experiencing this issue utilize Logitech Options (special button support e.g. gestures for space switching) or any other background process with Accessibility permissions (under System Preferences, Security & Privacy) granted to it?  I have confirmation that for me it's related to my use of Terminal full-screen and gestures to switch spaces; Logitech Options itself may never be "releasing the key" as it emulates ⌃← and ⌃→.  If I don't switch spaces after activation with a gesture, there is no problem.  Intriguing.
